### PR TITLE
Update whitelistApis.ts

### PR DESCRIPTION
**Pull request by ravisaurav-tarento**
bulkupdate changes, pls review

### CODE REVIEW

File: src/utils/whitelistApis.ts [hunk 1]
```diff
@@ -2833,6 +2833,20 @@ export const API_LIST = {
                 ROLE.MDO_ADMIN,
              ],
          },
+         '/proxies/v8/workflow/admin/transition/bulkupdate': {
+            checksNeeded: [CHECK.ROLE],
+            // tslint:disable-next-line: object-literal-sort-keys
+            ROLE_CHECK: [
+                ROLE.MDO_ADMIN,
```
Comment (karthik-tarento) [inline src/utils/whitelistApis.ts:2841]: MDO_LEADER also should have these functionalities... add that role too.

File: src/utils/whitelistApis.ts [hunk 2]
```diff
@@ -2833,6 +2833,20 @@ export const API_LIST = {
...
+             ],
+         },
+         '/proxies/v8/workflow/admin/bulkuploadfile/download/:fileName': {
+            checksNeeded: [CHECK.ROLE],
+            // tslint:disable-next-line: object-literal-sort-keys
+            ROLE_CHECK: [
+                ROLE.MDO_ADMIN,
```
Comment (karthik-tarento) [inline src/utils/whitelistApis.ts:2849]: MDO_LEADER also should have these functionalities... add that role too.

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

File: src/utils/whitelistApis.ts (hunk 1 above)
Comment (ravisaurav-tarento) [reply src/utils/whitelistApis.ts:2841]: This is done.

File: src/utils/whitelistApis.ts (hunk 2 above)
Comment (ravisaurav-tarento) [reply src/utils/whitelistApis.ts:2849]: This is done.

**Pull request merged**